phoenicis-dist/src/resources/Phoenicis.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Encoding=UTF-8
+Name=Phoenicis
+GenericName=Game
+Comment=Phoenicis
+Type=Application
+Exec=phoenicis %F
+Icon=phoenicis
+Categories=Game;
```

Comment:
Add newline at EOF.

Reply:
I think it is now updated isn't it?

Reply:
No. Check the diff. It shows this red symbol.

Reply:
Does not seem so: https://github.com/PhoenicisOrg/phoenicis/pull/1670/files

Reply:
Ok. Fixed.